zanata-war/src/main/java/org/zanata/security/permission/PermissionGranter.java

```diff
@@ -57,10 +59,7 @@ public PermissionGranter(Method granterMethod) {
      * Validates that the permission granter satisfies all preconditions.
      */
     public void validate() {
-        if (!Modifier.isStatic(granterMethod.getModifiers())) {
-            throw new RuntimeException("Permission method "
-                    + granterMethod.getName() + " must be static");
```

Comment:
I don't see why. Static methods will still work if someone chooses to write such a method. What do you think it gives us?